src/cli/json.rs

```diff
@@ -0,0 +1,555 @@
+use crate::prelude::{BTreeMap, String, Vec};
+
+use crate::errors;
+use core::convert::From;
+use rjson::{Array, Null, Object, Value};
+
+#[derive(PartialEq)]
+pub enum JsonValue {
```

Comment:
I think essentially this whole file could be deleted if you were using `serde_json`

Reply:
yes, and it is now removed 👍